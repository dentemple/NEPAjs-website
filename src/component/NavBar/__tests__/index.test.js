// @flow

import React from "react";
import { shallow, render } from "enzyme";
import { MemoryRouter } from "react-router-dom";
import MyComponent from "../index";

it("renders without crashing", () => {
  shallow(
    <MemoryRouter>
      <MyComponent />
    </MemoryRouter>
  );
});

it("matches the snapshot", () => {
  const component = render(
    <MemoryRouter>
      <MyComponent />
    </MemoryRouter>
  );
  expect(component).toMatchSnapshot();
});
