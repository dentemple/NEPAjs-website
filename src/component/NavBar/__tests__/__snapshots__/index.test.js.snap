// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {
          "class": "sc-bdVaJa bbCZxe",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "sc-bwzfXH dHjMNw",
            },
            "children": Array [
              Object {
                "attribs": Object {},
                "children": Array [
                  Object {
                    "attribs": Object {
                      "href": "/",
                    },
                    "children": Array [
                      Object {
                        "attribs": Object {
                          "class": "sc-htpNat dHjJs",
                          "src": "nepajs-logo.png",
                        },
                        "children": Array [],
                        "name": "img",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "tag",
                      },
                    ],
                    "name": "a",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                  },
                ],
                "name": "li",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
              },
            ],
            "name": "ul",
            "next": Object {
              "attribs": Object {
                "class": "sc-bwzfXH dHjMNw",
              },
              "children": Array [
                Object {
                  "attribs": Object {},
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "href": "http://www.tecbridgepa.org/",
                      },
                      "children": Array [
                        Object {
                          "attribs": Object {
                            "class": "sc-htpNat dHjJs",
                            "src": "tecbridge-logo.png",
                          },
                          "children": Array [],
                          "name": "img",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "tag",
                        },
                      ],
                      "name": "a",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                    },
                  ],
                  "name": "li",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                },
              ],
              "name": "ul",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
          },
          Object {
            "attribs": Object {
              "class": "sc-bwzfXH dHjMNw",
            },
            "children": Array [
              Object {
                "attribs": Object {},
                "children": Array [
                  Object {
                    "attribs": Object {
                      "href": "http://www.tecbridgepa.org/",
                    },
                    "children": Array [
                      Object {
                        "attribs": Object {
                          "class": "sc-htpNat dHjJs",
                          "src": "tecbridge-logo.png",
                        },
                        "children": Array [],
                        "name": "img",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "tag",
                      },
                    ],
                    "name": "a",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                  },
                ],
                "name": "li",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
              },
            ],
            "name": "ul",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "sc-bwzfXH dHjMNw",
              },
              "children": Array [
                Object {
                  "attribs": Object {},
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "href": "/",
                      },
                      "children": Array [
                        Object {
                          "attribs": Object {
                            "class": "sc-htpNat dHjJs",
                            "src": "nepajs-logo.png",
                          },
                          "children": Array [],
                          "name": "img",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "tag",
                        },
                      ],
                      "name": "a",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                    },
                  ],
                  "name": "li",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                },
              ],
              "name": "ul",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
            },
            "type": "tag",
          },
        ],
        "name": "nav",
        "next": null,
        "parent": null,
        "prev": null,
        "root": [Circular],
        "type": "tag",
      },
    ],
    "name": "root",
    "next": null,
    "parent": null,
    "prev": null,
    "type": "root",
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xmlMode": false,
  },
}
`;
